packages/parole/build.sh

```diff
@@ -6,5 +6,5 @@ TERMUX_PKG_VERSION=4.16.0
 TERMUX_PKG_SRCURL=http://deb.debian.org/debian/pool/main/p/parole/parole_${TERMUX_PKG_VERSION}.orig.tar.bz2
```

Comment:
Need to bump revision. *[CONTRIBUTING.md](https://github.com/termux/x11-packages/blob/master/CONTRIBUTING.md#rebuilding-package-with-no-version-change) already tells us to bump revision when rebuilding package, isn't it?*